cmd/limactl/protect.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/lima-vm/lima/pkg/store"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newProtectCommand() *cobra.Command {
+	var protectCommand = &cobra.Command{
+		Use:   "protect INSTANCE [INSTANCE, ...]",
+		Short: "Protect an instance to prohibit accidental removal",
+		Long: `Protect an instance to prohibit accidental removal via the 'limactl delete' command.
+The instance is not being protected agains removal via '/bin/rm', Finder, etc.`,
+		Args:              WrapArgsError(cobra.MinimumNArgs(1)),
+		RunE:              protectAction,
+		ValidArgsFunction: protectBashComplete,
+	}
+	return protectCommand
+}
+
+func protectAction(_ *cobra.Command, args []string) error {
+	var errs []error
+	for _, instName := range args {
+		inst, err := store.Inspect(instName)
+		if err != nil {
+			errs = append(errs, fmt.Errorf("failed to inspect instance %q: %w", instName, err))
+			continue
+		}
+		if err := inst.Protect(); err != nil {
```

Comment:
I think we should check if the instance is already protected, and print a friendly message like `Instance %q is already protected` instead of this error:

```
$ l protect alpine
FATA[0000] failed to protect instance "alpine": open /Users/jan/.lima/alpine/protected: permission denied
```

Reply:
Fixed